src/Tooltip/index.jsx

```diff
@@ -68,35 +77,50 @@ export default class Tooltip extends Component
             cssMap,
             message,
             position,
+            isDismissible,
             isVisible,
             noWrap,
+            onClickClose,
             onMouseOver,
             onMouseOut,
             overflowIsHidden
         } = this.props;
 
         const { id } = this.state;
 
-        let messageLines = message;
+        let messageText = message;
 
-        if ( typeof message === 'string' )
+        if ( typeof messageText === 'string' )
         {
             const regexp = /([^\s].{1,49}(?=\s+|$))|([^\s]{1,50})/g;
 
             const matches = message.match( regexp ) || [];
 
-            messageLines = matches.map( ( line, index ) =>
-                // eslint-disable-next-line react/no-array-index-key
-                <div key = { index }>{line}</div> );
+            messageText = ( <Text
+                className = { cssMap.message }
+                noWrap >
+                { matches.map( line => [ line, <br /> ] ) }
+            </Text> );
         }
 
+
         const tooltip = (
             <div
-                className = { cssMap.tooltip }
+                className = { isDismissible ? cssMap.tooltipDismissible :
+                    cssMap.tooltip }
                 role      = "tooltip"
                 id        = { id }>
-                <div
-                    className = { cssMap.message }>{ messageLines }</div>
+                <div className = { cssMap.messageContainer }>
```

Comment:
For the sake of CSS backwards compatibility with Sociomantic legacy Backbone-Marionette implementation I think we should change this classname back to just `message`.

(Would need to be changed back in the test driver too.)